v0.5/tools/submission/submission-checker.py

```diff
@@ -26,6 +26,13 @@
 REQUIRED_PERF_FILES = ["mlperf_log_accuracy.json", "mlperf_log_summary.txt", "mlperf_log_detail.txt"]
 REQUIRED_ACC_FILES = REQUIRED_PERF_FILES + ["accuracy.txt"]
 REQUIRED_MEASURE_FILES = ["mlperf.conf", "user.conf", "README.md"]
+PERFORMANCE_SAMPLE_COUNT = {
+    "mobilenet": 1024,
+    "resnet50": 1024,
+    "ssd-mobilenet": 256,
+    "ssd-resnet34": 64,
+    "gnmt": 3003,
```

Comment:
Should this be 3003 or 3903900? @DilipSequeira @christ1ne

Reply:
Yes, https://github.com/mlperf/inference_policies/blob/master/inference_rules.adoc#41-benchmarks wants 3903900 ... is this correct ?

Reply:
I changed it to 3903900 to align with the rules doc.